features/userprofile/shared/src/main/kotlin/io/element/android/features/userprofile/shared/UserProfileView.kt

```diff
@@ -51,7 +50,6 @@ fun UserProfileView(
     openAvatarPreview: (username: String, url: String) -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    BackHandler { goBack() }
```

Comment:
I have checked that this removal does not break the "physical" back button press and this is fine because this Node is added to a `BackStack`.